src/physics/physics.cpp

```diff
@@ -197,6 +202,24 @@ ::post_get_needed_update_flags () const
     return dealii::update_values;
 }
 
+template <int dim, int nstate, typename real>
+template<typename real2>
+real2 PhysicsBase<dim,nstate,real>
+::handle_non_physical_result() const
+{
+    if (this->non_physical_behavior_type == NonPhysicalBehaviorEnum::abort_run) {
+        this->pcout << "ERROR: Non-physical result has been detected. Aborting... " << std::endl << std::flush;
+        std::abort();
+    } else if (this->non_physical_behavior_type == NonPhysicalBehaviorEnum::print_warning) {
+        this->pcout << "WARNING: Non-physical result has been detected at a node." << std::endl;
+    } else if (this->non_physical_behavior_type == NonPhysicalBehaviorEnum::do_nothing) {
```

Comment:
maybe `set_to_big_number` instead of `do_nothing` ?

Reply:
Sure, that is more clear. Updated.